en/config_heli/README.md

```diff
@@ -0,0 +1,87 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
+We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
+:::
+
+## Supported Configurations
+
+Supported helicopter configurations:
+- Single main rotor with any type of swash-plate
+- ESC tail rotor
+
+
+## Setup
+
+To setup and configure a helicopter: 
+
+1. Select the helicopter [Airframe](../config/airframe.md) in QGroundControl
+1. Configure [Actuator](../config/actuators.md) geometry
+   ![Geometry helicopter](../../assets/config/actuators/qgc_geometry_helicopter.png)
+   1. Remove the rotor blades and propellers
+   1. Configure the swash-plate servos.
+      The angle is measured clockwise with 0 pointing forwards.
+      Trim is typically not needed, but can be used to offset individual servo positions.
+   1. Set the main rotor turning orientation
```

Comment:
I guess options are disabled/enabled?

Reply:
As you copied these over to the actuator page, did you still want to keep this here?

Reply:
Good point. Removed this bit.